backend/views/views.py

```diff
@@ -293,3 +266,16 @@ def restore_issue(issue_id):
     issue.restore()
     db.session.commit()
     return redirect(url_for('issues_page'))
+
+
+@app.route('/issue/<int:issue_id>', methods=['GET'])
+@admin_permissions
+def issue(issue_id):
```

Comment:
change name of function "issue", because of such variable exists and pylint throw warning
W:237, 4: Redefining name 'issue' from outer scope (line 273) (redefined-outer-name)
W:255, 4: Redefining name 'issue' from outer scope (line 273) (redefined-outer-name)
W:265, 4: Redefining name 'issue' from outer scope (line 273) (redefined-outer-name)